docs/public-networks/concepts/sync-node.md

```diff
@@ -1,18 +1,18 @@
 ---
-title: Sync Besu
-sidebar_position: 1
-description: Full and archive node types
+title: Node synchronization
+sidebar_position: 10
+description: Sync full and archive node types.
 tags:
   - public networks
 ---
 
-# Sync Besu
+# Sync nodes
```

Comment:
After reviewing this page, I'm wondering if we should take this opportunity to restructure it more intuitively for a "Concepts" page. My suggested restructure:

- Node synchronization
  - Node types
    - Full nodes
    - Archive nodes
  - Sync modes
    - Snap sync
    - Checkpoint sync
    - Fast sync
    - Full sync
  - Sync times
  - Storage

Reply:
May we open another PR for this? I have had the PR open for while and I think this promotes a bit of scope creep.

Reply:
That's fine with me 👍